deploy/test-environments/fleet_api/src/agents_enrolled.py

```diff
@@ -0,0 +1,81 @@
+"""
+Wait for agents to be enrolled for a given policies
+If the expected number of agents is not enrolled within the timeout, the test will fail
+"""
+import sys
+import time
+from api.agent_policy_api import get_agents
+import configuration_fleet as cnfg
+from state_file_manager import state_manager
+from loguru import logger
+
+TIMEOUT = 600
+
+
+def get_expected_agents_mapping() -> dict:
```

Comment:
no need to add the return type in the function name if it's type, kind of confusing since agent mapping sound like a term, and wasn't sure what are you referring to